dsl/src/main/java/cd/go/contrib/plugins/configrepo/groovy/dsl/DependencyMaterial.java

```diff
@@ -44,6 +44,9 @@
     @NotEmpty
     private String stage;
 
+    @JsonProperty("ignore_for_scheduling")
```

Comment:
@akshaydewan: do you mind adding some javadoc here, once you update the docs on the main site/